src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -158,6 +158,7 @@
         <service id="sulu_media.adapter.gd" class="Imagine\Gd\Imagine" public="false"/>
         <service id="sulu_media.adapter.imagick" class="Imagine\Imagick\Imagine" public="false"/>
         <service id="sulu_media.adapter.gmagick" class="Imagine\Gmagick\Imagine" public="false"/>
+        <service id="sulu_media.adapter.vips" class="Imagine\Vips\Imagine" public="false"/>
```

Comment:
@wachterjohannes this should be removed from this file

Reply:
done